server/db/models/user.js

```diff
@@ -6,10 +6,13 @@ const User = db.define('user', {
   email: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
+    allowNull: false,
+    validate: {
+      isEmail: true
+    }
   },
   password: {
-    type: Sequelize.STRING
+    type: Sequelize.STRING,
```

Comment:
notEmpty for password.

Reply:
Concerned about facebook and google login -- is a password created with Oauth?

Reply:
I'm following the Oauth workshop -- and not requiring a password if they sign in with google or facebook. 
We could create virtual setter that puts their facebook or googleID in the password, if they sign  in with google or facebook. But that my be overkill.